src/Utilities/matrix_of_constraints.jl

```diff
@@ -124,10 +121,10 @@ mutable struct MatrixOfConstraints{T,AT,BT,ST} <: MOI.ModelLike
     coefficients::AT
     constants::BT
     sets::ST
-    are_indices_mapped::BitSet
-    caches::Union{Nothing,Vector}
+    caches::Vector
```

Comment:
`Vector{Any}`? Or `Vector{Vector{Tuple{DataType,DataType}}}`?

Reply:
Each entry of `caches` is a `Vector` of concrete `eltype`. If we write `Vector{Vector{...}}` where `...` is an abstract type then it will convert these vectors to vector with non-concrete eltypes.